src/iosurface.cs

```diff
@@ -228,5 +228,13 @@ interface IOSurface : NSSecureCoding
 		[iOS (11,0)][Mac (10,13)]
 		[Internal, Export ("setPurgeable:oldState:")]
 		int _SetPurgeable (IOSurfacePurgeabilityState newState, IntPtr oldStatePtr);
+
+		[Watch (4,0), TV (11,0), Mac (10,6), iOS (11,0), MacCatalyst (13,0)]
+		[Field ("kIOSurfaceColorSpace")]
+		NSString kIOSurfaceColorSpace { get; }
```

Comment:
kIOSurfaceColorSpace is not correct. Those are global in objc and that is why they add the framework name and also that k which stands for key.